lib/build-tokens.js

```diff
@@ -154,13 +154,13 @@ async function buildTokensCommand(commandArgs) {
   ];
 
   // Build tokens for each configuration
-  await Promise.all(configs.map(async ({ config, isThemeVariant, themeVariant }) => {
+  await Promise.all(configs.map(async ({ config, themeVariant }) => {
     const sd = new StyleDictionary(config);
     await sd.cleanAllPlatforms();
     await sd.buildAllPlatforms();
     createIndexCssFile({
       buildDir,
-      isThemeVariant: !!isThemeVariant,
+      isThemeVariant: !!themeVariant,
```

Comment:
I noticed that `isThemeVariant` always has the value `undefined` and in `styles/css/themes/light` the `index.css` file is not generated